source/dashboards/views.markdown

```diff
@@ -73,7 +73,7 @@ views:
       ...
 ```
 
-### Options For Visible
+### Options for visible
```

Comment:
```suggestion
### Options for `visible`
```
As this is a YAML key option, should we here also use backticks?

For the UI configuration of it, it ss named "visibility"

Reply:
This comment applies to all of these options not just `visible `

Reply:
added this, then, after a second look, removed it again. I prefer to keep the code fence out of the title to focus more on the content of the topic.